### ISSUE TITLE
a11y_Input> Clickable Icon Large> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#500]
**Describe the bug**
Input>> "1 violations" is appearing for "Clickable Icon Large". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/input--clickable-icon-large'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for Input> Clickable Icon Large.

**Screenshots**
https://somup.com/c3n2jCZW8w

**Desktop :**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The issue is fixed:
See Screen-cast : http://somup.com/c3jDelu8yE
Thanks!

**Comment by Uditsingh67** [comment]
Issue does not exist.
See Screen-cast : http://somup.com/c3jwhBU2qj
